addons/website/tests/test_website_visitor.py

```diff
@@ -1,13 +1,16 @@
 # coding: utf-8
-from odoo.tests import HttpCase
+from odoo import tests
+from datetime import datetime, timedelta
 
 
-class WebsiteVisitorTests(HttpCase):
-    def test_create_visitor_on_tracked_page(self):
+@tests.tagged('dbetest')
+class WebsiteVisitorTests(tests.HttpCase):
+    def setUp(self):
+        super().setUp()
```

Comment:
Is this a valid way to call super?

I've always seen `super(WebsiteVisitorTests, self).setUp()`